checks/automatic_dependency_update.go

```diff
@@ -29,23 +29,28 @@ func init() {
 
 // AutomaticDependencyUpdate will check the repository if it contains Automatic dependency update.
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged

Reply:
You can actually set the PR to be merged into your dependent branch, then changes will show relative to that instead. 

Once the dependent branch is merged, this one will automatically rebase back to main.